pkg/storage/bloom/v1/util.go

```diff
@@ -44,24 +44,49 @@ var (
 
 	// buffer pool for bloom pages
 	// 128KB 256KB 512KB 1MB 2MB 4MB 8MB 16MB 32MB 64MB 128MB
-	BloomPagePool = BytePool{
+	BloomPagePool = &BytePool{
```

Comment:
IIUC we will no longer be using this one but we are still allocating it. Should we remove it?

Reply:
We use it as default Allocator type, if you don't specify anything in the config. It's the type "dynamic" (yeah, rather bad name, I know ...)